doc/POINTS_COORDINATES.md

```diff
@@ -0,0 +1,169 @@
+# Points, tripoints, and coordinate systems
+
+## Coordinate systems
+
+CDDA uses a variety of coordinate systems for different purposes.  These differ
+by scale and origin.
+
+The most precise coordinates are **map square** (ms) coordinates.  These refer to
+the tiles you see normally when playing the game.
+
+Two origins for map square coordinates are common:
```

Comment:
aren't absolute and local also relevant to overmap terrain coordinates?

Absolute is also inconsistently aliased to global, and I've never been clear if there is a difference between the two terms.

Reply:
I don't think there's such a thing as local overmap terrain coordinates.  There are overmap-relative overmap terrain coordinates, but they're not referred to as local.

Yeah, the absolute vs. global thing is a minor annoyance.  Especially because some vehicle code uses global to refer to non-vehicle local coordinates.  I will mention global here, but we should probably try to be more consistent.